### PR TITLE
Chore: Improve test coverage and implement minimum coverage requirements

### PR DESCRIPTION
**Pull request by erdimaden**
### What changed? Why?

- Added test cases to enhance PR coverage.
- Set minimum test coverage requirements:
  - Branches: 80%
  - Functions: 90%
  - Statements: 95%
  - Lines: 95%
- Moved test files to the main folder for better organization.

### CODE REVIEW

File: src/tests/config/coinbase_cloud_api_key.json
Comment (alex-stone) [inline src/tests/config/coinbase_cloud_api_key.json:1]: Is this a valid API Key? Want to make sure we're not exposing an API key that anyone who can see this repo can just use?

Assuming it is a test key that isn't usable, can we rename this something like `test_api_key.json` to make that clear?

Comment (erdimaden) [reply src/tests/config/coinbase_cloud_api_key.json:1]: Yes, it is not a valid API Key. I updated to `test_api_key.json`

File: src/tests/transfer_test.ts [hunk 1]
```diff
@@ -114,7 +126,15 @@ describe("Transfer Class", () => {
     });
   });
 
-  describe("getTransaction", () => {
+  describe("#getTransactionLink", () => {
+    it("should return the transaction link when the transaction hash is available", () => {
+      expect(transfer.getTransactionLink()).toEqual(
+        `https://sepolia.basescan.org/tx/${transferModel.transaction_hash}`,
```
Comment (alex-stone) [inline src/tests/transfer_test.ts:132]: Should we use the transaction link property on the transfer model?

https://github.com/coinbase/coinbase-sdk-nodejs/blob/aaaf99926847e316b8cf51c0699b2971fb1548b8/src/coinbase/transaction.ts#L101
https://github.com/coinbase/coinbase-sdk-ruby/blob/c836dedacd7c337b4f6cd85f00fd049562cad79a/lib/coinbase/transaction.rb#L78-L80

Can land in follow-up?

**Review by alex-stone — APPROVED** [review]

File: src/tests/transfer_test.ts (hunk 1 above)
Comment (erdimaden) [reply src/tests/transfer_test.ts:132]: Sure, I'm going to fix that in my next PR 👍

**Pull request merged**